.github/workflows/switch.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           rm stk-assets-full.zip
       - name: Restore modified date
         run: |
-          sudo apt install -y git-restore-mtime
+          sudo apt update && sudo install -y git-restore-mtime
```

Comment:
> `sudo install`

Typo? I think this should have been `sudo apt install`

Reply:
You're right

Reply:
Oops, clearly I wasn't paying attention. I've added git-restore-mtime to the docker image now though so this isn't needed any more.